data/zilliqa/assets/zil1jy3g5j9w5njqwxuuv3zwkz9syyueelmu7g080v/meta.json

```diff
@@ -0,0 +1,19 @@
+{
+ "name": "Unifees", 
+ "symbol": "FEES", 
+ "web": "https://unifees.io",
+ "supply": "1000000000", 
+ "links": { 
+ "telegram": "https://t.me/unifees", 
+ "linkedin": "https://www.linkedin.com/company/unifees", 
+ "twitter": "https://twitter.com/UnifeesToken", 
+ "github": "https://github.com/Unifees/Unifees-Token", 
+ "instagram": "https://www.instagram.com/unifeestoken", 
+ "facebook": "https://www.facebook.com/unifeestoken" 
+ }, 
+ "product": true, 
```

Comment:
remove

Reply:
Hello , is there something wrong with the file ? Please let me know

Reply:
I commented on the line that needs to be removed

Reply:
Removed the commented line

Reply:
you didn't

Reply:
@balthazar I have removed the lines in commit id **c5a44a0** commit message **removed instafaceboo**

https://github.com/Ashlar/cryptometa/pull/236/commits/c5a44a0d0b98e38b575d85351160399bb731eeac